static/js/apps/explore_landing/topics.json

```diff
@@ -1,15 +1,42 @@
 {
   "allTopics": [
-    "economy",
-    "health",
-    "education",
-    "environment",
+    "equity",
+    "sustainability",
     "demographics",
-    "sustainability"
+    "economics",
+    "schools-housing-commute",
+    "health"
   ],
   "topics": {
-    "economy": {
-      "title": "Economy",
+    "equity": {
+      "title": "Equity",
+      "description": "Equity data serves as a vital compass in our pursuit of a just and inclusive society, offering crucial insights into disparities, privileges, and opportunities among diverse groups. Understanding equity empowers us to identify and address systemic imbalances, fostering a world where every individual can thrive and contribute to a shared and equitable future. Search for some equity data or click around on one of the suggested searches below to get started.",
+      "examples": {
+        "general": [
+          {
+            "title": "Economic equity in California",
+            "url": "oq=Economic+equity+in+California"
+          },
+          {
+            "title": "Incarceration rates by race in the US",
+            "url": "oq=Incarceration+rates+by+race+in+the+US"
+          },
```

Comment:
We have topics for these so we can append stuff like:

&t=dc/topic/EconomicEquity&p=geoId/06
&t=dc/topic/JusticeSystemEquity&p=country/USA
&t=dc/topic/HealthEquity&p=geoId/06085

Reply:
added